File: src/Page/SelectedMyCoursePage.java
package Page;

import SystemMessage.FunctionType;
import SystemMessage.SystemMessage;

public class SelectedMyCoursePage extends Page{
	
	public SelectedMyCoursePage(String content) {
		super(content);
		setType(PageType.SELECTED_MY_COURSE_PAGE);
		setName("Selected My Course Page");
	}
	
	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MY_COURSES_PAGE, null);
	}
}

File: iter3/python_development/Page/SelectedMyCoursePage.py
from iter3.python_development.Page.Page import Page
from iter3.python_development.Page.PageType import PageType
from iter3.python_development.System.FunctionType import FunctionType
from iter3.python_development.System.SystemMessage import SystemMessage


class SelectedMyCoursePage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType("SELECTED_MY_COURSE_PAGE")
        self.setName("Selected My Course Page")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage("CHANGE_PAGE", "MY_COURSES_PAGE", None)
